shared/common-adapters/icon.constants.js

```diff
@@ -13,197 +13,190 @@ const iconMeta_ = {
   'iconfont-caret-down': {
     isFont: true,
     gridSize: 8,
-    charCode: 59648,
```

Comment:
`charCode` got removed from every icon.constants.js entry.. ah, maybe the CSS formatting change at icomoon's end broke our npm-updated-fonts.  I can't see any breakage in the app caused by this but I'll look into why that's missing now.